thirdparty/github.com/go-yaml/yaml.v2/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
It's good to include the license, but I don't think this bubbles up to the root and gets picked up by dependents like we need it to. We may need to include this in the root LICENSE file as well

Reply:
Do I just need to append the contents of this file to the root LICENSE?

Reply:
Based on conversation, I updated the root license by appending the forked yaml libs' licenses' content to the root license, hopefully that is enough for now.